docs/content/preview/migrate/reference/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Reference
+headerTitle: Reference
+linkTitle: Reference
+image: /images/section_icons/develop/learn.png
+headcontent: Command line interfaces (CLIs), data modeling strategies and data type mapping reference.
+type: indexpage
+showRightNav: true
+menu:
+  preview:
+    identifier: reference-voyager
+    parent: voyager
+---
```

Comment:
known issues should prob go last in leftnav; or possibly under reference

Reply:
Will address this when I restructure known issues.